pkg/prompt/prompt_test.go

```diff
@@ -0,0 +1,27 @@
+package prompt
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetInput(t *testing.T) {
+	tests := []struct {
+		v string
+		e bool
+	}{
+		{v: "hello\n", e: false},
```

Comment:
since you're specifying all the fields in the right order, you can leave off the field names here. they are close enough to the struct definition that I don't think it hurts readability.

Reply:
Done